examples/stable-diffusion/training/train_text_to_image_sdxl.py

```diff
@@ -519,19 +474,43 @@ def parse_args(input_args=None):
             " lazy mode."
         ),
     )
-    parser.add_argument("--use_hpu_graphs", action="store_true", help="Use HPU graphs on HPU.")
     parser.add_argument(
-        "--rank",
-        type=int,
-        default=4,
-        help=("The dimension of the LoRA update matrices."),
+        "--use_hpu_graphs_for_training", action="store_true", help="Use HPU graphs for training on HPU."
     )
     parser.add_argument(
-        "--finetuning_method",
+        "--use_hpu_graphs_for_inference", action="store_true", help="Use HPU graphs for inference on HPU."
+    )
+
+    parser.add_argument(
+        "--image_save_dir",
         type=str,
-        default="default",
-        choices=["default", "lora"],
-        help=("Set the method for fine-tuning." " Choices: ['default', 'lora']."),
+        default="./stable-diffusion-generated-images",
+        help="The directory where images will be saved.",
+    )
+    parser.add_argument(
+        "--output_type",
+        type=str,
+        choices=["pil", "np"],
+        default="pil",
+        help="Whether to return PIL images or Numpy arrays.",
+    )
+    parser.add_argument(
+        "--profiling_warmup_steps",
+        default=0,
+        type=int,
+        help="Number of steps to ignore for profiling.",
+    )
+    parser.add_argument(
+        "--profiling_steps",
+        default=0,
+        type=int,
+        help="Number of steps to capture for profiling.",
+    )
+    parser.add_argument(
+        "--logging_step",
+        default=1,
```

Comment:
I think it depends on how many steps user will run. For instance, the total step is just 336 in 8x run.
It would be still ok as user can configure their logging with this argument.